aws/table_aws_securityhub_insight.go

```diff
@@ -0,0 +1,157 @@
+package aws
+
+import (
+	"context"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/securityhub"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsSecurityHubInsight(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_securityhub_insight",
+		Description: "AWS Securityhub Insight",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("insight_arn"),
+			Hydrate:           getSecurityHubInsight,
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFoundException", "InvalidInputException"}),
+		},
+		List: &plugin.ListConfig{
+			Hydrate:           listSecurityHubInsights,
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFoundException"}),
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of a Security Hub insight.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "insight_arn",
```

Comment:
```suggestion
				Name:        "arn",
```

We normalize the ARN columns to just `arn` so we can use them across mod controls, queries, etc.